Context/Adapter/JMSContextAdapter.php

```diff
@@ -74,7 +76,7 @@ public function convertDeserializationContext(ContextInterface $context)
      */
     public function supportsSerialization(ContextInterface $context)
     {
-        return $this->serializer instanceof JMSSerializerInterface;
+        return $this->serializer instanceof JmsSerializerAdapter;
```

Comment:
May be better to use the same vendor syntax as in [``jms/serializer``](https://github.com/schmittjoh/serializer/tree/master/src/JMS/Serializer):
``JMSSerializerAdapter``.